interp/cfg.go

```diff
@@ -3024,6 +2928,121 @@ func compositeGenerator(n *node, typ *itype, rtyp reflect.Type) (gen bltnGenerat
 	return gen
 }
 
+func matchSelectorMethod(sc *scope, n *node, name string) (err error) {
```

Comment:
The signature can be simplified, just like so:
```suggestion
func matchSelectorMethod(sc *scope, n *node) (err error) {
	name := n.child[1].ident
```